maplibre_gl/android/src/main/java/org/maplibre/maplibregl/LayerPropertyConverter.java

```diff
@@ -65,7 +74,16 @@ static PropertyValue[] interpretSymbolLayerProperties(Object o) {
           properties.add(PropertyFactory.textHaloBlur(expression));
           break;
         case "text-translate":
-          properties.add(PropertyFactory.textTranslate(expression));
+          if (jsonElement.isJsonArray()) {
```

Comment:
Maybe put all this logic into `convertJsonToFloatArray` to avoid repetition?..

Reply:
I actually don't remember why I did this, it was a while ago. Maybe because in some cases the conversion was handled differently, but currently it seems logical to put everything in `convertJsonToFloatArray`.